cruise.umple/test/cruise/umple/compiler/487_parallelStateMachines_diffRegionSameNames.ump

```diff
@@ -0,0 +1,27 @@
+class X {
```

Comment:
In my tests --> assertNoWarnings("487_parallelStateMachines_diffRegionSameNames.ump");